pkg/oci/utils.go

```diff
@@ -32,8 +32,8 @@ var (
 	// ErrNoLinux is an error for missing Linux sections in the OCI configuration file.
 	ErrNoLinux = errors.New("missing Linux section")
 
-	// ociConfigPathKey is the annotation key to fetch the OCI config.json file path.
-	ociConfigPathKey = "oci/config_path"
+	// ociConfigPathKey is the annotation key to fetch the OCI configuration file path.
+	ociConfigPathKey = "com.github.containers.virtcontainers.pkg.oci.config_path"
```

Comment:
I think you're talking about a different thing here. The OCI specs defines some annotations but our Annotations field in virtcontainers is not OCI specific. It is intended to provide any runtime implementation, a way to store extra data that are specific to the specification.
I think we should not merge this change.